packages/neo-one-smart-contract-compiler/src/compile/syscalls.ts

```diff
@@ -1208,4 +1208,73 @@ export const SYSCALLS = {
       },
     };
   })(),
+  'Neo.Runtime.Call': (() => {
+    const name = 'Neo.Runtime.Call';
+    const valueType = SerializableValue;
+    const returnType = valueType;
+
+    return {
+      name,
+      toDeclaration(): string {
+        const sig = valueType.toSignature();
+
+        return `function syscall(name: '${name}', hash: Buffer, ...args: Array<${sig}>): ${returnType.toSignature()};`;
+      },
+      handleCall(sb: ScriptBuilder, node: CallExpression, optionsIn: VisitOptions): void {
+        const options = sb.pushValueOptions(sb.noCastOptions(optionsIn));
+
+        // exprCall  = Buffer.from("1225ACFA1", "hex"),
+        const exprCall = node.getArguments()[1];
+
+        if (!TypeGuards.isCallExpression(exprCall)) {
+          throw new Error('ARG[1] Expecting callExpression ');
+        }
+
+        // exprCallObj = [Buffer.from]
+        const exprCallObj = exprCall.getExpression();
+
+        if (!TypeGuards.isPropertyAccessExpression(exprCallObj)) {
+          throw new Error('Expected: Property Access expression');
+        }
+
+        // exprCallObjName = "Buffer"
+        const exprCallObjName = exprCallObj.getExpression();
+
+        // exprCallObjFnx = "from"
+        const exprCallObjFnxName = exprCallObj.getName();
+
+        if (
+          !(
+            TypeGuards.isIdentifier(exprCallObjName) &&
+            sb.isGlobal(exprCallObjName, sb.getType(exprCallObjName), 'Buffer')
+          ) ||
+          exprCallObjFnxName !== 'from'
+        ) {
+          throw new Error("Expected 'Buffer.from' expression");
+        }
+
+        const argContractId = exprCall.getArguments()[0];
+        if (!TypeGuards.isStringLiteral(argContractId)) {
+          const argContractIdText = exprCall.getText();
+          throw new Error(`Expected contract identifier with Neo.Runtime.CallContract but got: ${argContractIdText}`);
```

Comment:
Just put `expr.getText()` directly in the string rather than making another variable

Reply:
Same comment, put `expr.getText()` directly in the template literal.